package/apt/test.sh

```diff
@@ -22,14 +22,15 @@ sudo apt install -V -y curl lsb-release
 
 code_name=$(lsb_release --codename --short)
 architecture=$(dpkg --print-architecture)
+distribution=$(lsb_release --short --id | tr 'A-Z' 'a-z')
```

Comment:
Distribution, code_name and architecture are normal order such as "Ubuntu Jammy amd64".

```suggestion
distribution=$(lsb_release --short --id | tr 'A-Z' 'a-z')
code_name=$(lsb_release --codename --short)
architecture=$(dpkg --print-architecture)
```

Reply:
Thanks!